src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -119,6 +119,7 @@ export const SimplePowerBIReportEmbed = ({
     () => PowerBIUtils.constructDynamicFilters(dynamicFilters, scenarioDTO),
     [dynamicFilters, scenarioDTO]
   );
+  const resultsDisplayDisabled = reports?.status === 'DISABLED';
```

Comment:
I'm not sure about the case of the constant value. As below, we use simple lower cases to set a value. I think full higher cases is reserved for const name itself.

Reply:
as it's a value set in Redux and all store statuses use upper case, like ERROR on line 235, so I choose to do the same for this status :slightly_smiling_face: